lua/nvim-tree/fs.lua

```diff
@@ -317,4 +317,32 @@ function M.print_clipboard()
   return api.nvim_out_write(table.concat(content, '\n')..'\n')
 end
 
+local function copy_to_clipboard(content)
+  vim.fn.setreg('+', content);
+  vim.fn.setreg('"', content);
+  return api.nvim_out_write(string.format('Copy %s to system clipboard! \n', content))
```

Comment:
`Copied` would be more appropriate here

Reply:
Okay, I renamed this function name to `Copied` .